CHANGELOG.unreleased.md

```diff
@@ -8,3 +8,4 @@
 
 - xo-server v5.41.0
 - xo-web v5.41.0
+- @xen-orchestra/fs v0.9.0
```

Comment:
Packages are released in order from top to bottom, therefore they should be listed in reverse-dependency order.

Rule of thumb: always put new packages at the top of the list.